core/__tests__/models/destination/modelCache.ts

```diff
@@ -112,7 +112,8 @@ describe("models/destinationsCache", () => {
 
     test("after a destination is updated, the local cache should be invalid", async () => {
       DestinationsCache.expires = new Date().getTime() + 1000 * 30;
-      await destination.update({ key: "LAST NAME" });
+      await destination.update({ name: "NEW NAME" });
```

Comment:
Was this change intentional?

Reply:
Yep! Destinations don't have keys... they do have names though